shepherd_bms/src/compute.cpp

```diff
@@ -17,7 +17,7 @@ void ComputeInterface::enableCharging(bool enableCharging)
     isChargingEnabled = enableCharging;
 }
 
-FaultStatus_t ComputeInterface::sendChargingMessage(uint16_t voltageToSet, uint16_t currentToSet)
+FaultStatus_t ComputeInterface::sendChargingMessage(uint16_t voltageToSet, uint16_t currentToSet, uint8_t state_of_charge, AccumulatorData_t *bms_data)
```

Comment:
```suggestion
FaultStatus_t ComputeInterface::sendChargingMessage(uint16_t voltageToSet, uint16_t currentToSet, AccumulatorData_t *bms_data)
```
I think we're good to remove this now right? (+ remove from header file and call to function)

Reply:
whoops didnt realize we called that in the telemetry broadcast, yes that can be removed ill do now